src/test/java/com/groupbyinc/api/QueryTest.java

```diff
@@ -576,4 +576,20 @@ public void testPinnedRefinementsWithoutValueRefinements() {
                 "    'skip': 0" + //
                 "}", test);
   }
+
+  @Test
+  public void testPreFilterExpression() {
+    test.setQuery("nice products");
+    test.setPreFilterExpression("brand = \"shiny\"");
```

Comment:
Another side question, do we support '  or just
i.e. "brand = 'shiny'"
vs "brand = \"shiny\""

Reply:
Answering my own question.
From the API-javascript repos, I can see why we may just support the ".
In javascript, most people are use to surrounding literals with '
.withPreFilterExpression('brand = "shiny"')

Reply:
Yeah that's right. In the JavaScript project there are linting rules that only allow you to use single quotes for strings.